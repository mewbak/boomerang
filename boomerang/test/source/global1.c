
int a = 5;
int b = 7;

void foo2()
{
    b = 12;
    printf("a = %i\n", a);
}

void foo1()
{
    foo2();
}

int main()
{
    foo1();
    printf("b = %i\n", b);
    return 0;
}



