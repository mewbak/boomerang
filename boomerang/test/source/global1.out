a = 5
b = 12
